packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -297,6 +303,10 @@ $expand-less-icon: map-merge(
         padding-right: units(2);
       }
     }
+
+    &::after {
+      display: none;
+    }
```

Comment:
What's this for? Is it required?

Reply:
Nope! accidentally included. Thanks for catching!

Reply:
Resolved in [85d12c9](https://github.com/uswds/uswds/pull/5286/commits/85d12c980e65094cb35ee99896b6904305c77c7d)